node/service/src/rpc/mod.rs

```diff
@@ -82,7 +84,7 @@ where
 	module.merge(TransactionPayment::new(client.clone()).into_rpc())?;
```

Comment:
Can `client.clone()` be performed once?

```
let client_clone = client.clone();
module.merge(MessagesHandler::new(client_clone).into_rpc())?;
module.merge(SchemasHandler::new(client_clone).into_rpc())?;
```

Reply:
Unfortunately that is not how Rust works. In passing the client we are moving the instance inside the constructor so we need a clone. And since client is of type `Arc` cloning is not expensive at all.